src/api/stores.py

```diff
@@ -20,10 +21,14 @@ class StoreLocation(BaseModel):
     longitude: float = Field(le=180, ge=-180)
     latitude: float = Field(le=90, ge=-90)
 
+class Hours(BaseModel):
```

Comment:
could we instead of using strings maybe use four digit integers? So that 2:00 in the afternoon would be 1400? Or even two ints of hours and minutes, that way we can have checks if the times are valid.